mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -4351,10 +4433,16 @@ pub async fn create_maker_order(ctx: &MmArc, req: SetPriceReq) -> Result<MakerOr
         rel_coin.coin_protocol_info(),
     );
 
-    {
-        let mut my_maker_orders = ordermatch_ctx.my_maker_orders.lock();
-        my_maker_orders.insert(new_order.uuid, Arc::new(AsyncMutex::new(new_order.clone())));
+    let mut makerorders_ctx = ordermatch_ctx.makerorders_ctx.lock();
+    if !makerorders_ctx.balance_loop_exists(base_coin.ticker()) {
+        let ctx_weak = ctx.weak();
+        let ticker = base_coin.ticker().to_string();
+        let handle =
+            common::spawn_abortable(async move { check_balance_update_loop(ctx_weak, ticker, Some(balance)).await });
```

Comment:
Please import `spawn_abortable` instead of using the fully qualified path. I have mentioned `common::spawn_abortable` for easier search :) It's preferred to use imports in general, adding a path is reasonable when we need to handle multiple imports with the same name as here:
https://github.com/KomodoPlatform/atomicDEX-API/blob/b8598439a38c7e0e662ae2cce60ec1bd23162ef6/mm2src/coins/z_coin/z_coin_errors.rs#L197